src/main/java/hello/Application.java

```diff
@@ -0,0 +1,48 @@
+package hello;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
+public class Application implements CommandLineRunner {
+
+	@Autowired
+	private StudentRepository repository;
+
+	public static void main(String[] args) {
```

Comment:
Why are there two `main()s`?

Reply:
because one is for the repositories and one is for the greetings page (which has nothing to do with our project its just and example of creating views)